package.json

```
@@ -38,8 +38,8 @@
     "test:playwright:update-tapes": "UPDATE_TAPES=true pnpm test:playwright",
     "test:playwright:gen": "playwright codegen localhost:8443/",
     "types": "vue-tsc --noEmit",
-    "lint": "eslint --ext .js,.vue,.ts --ignore-path .gitignore --ignore-path .eslintignore . && prettier --check .",
-    "lint:fix": "pnpm lint -- --fix && prettier --write --loglevel=warn .",
+    "lint": "prettier --check .; eslint --ext .js,.vue,.ts --ignore-path .gitignore --ignore-path .eslintignore .",
+    "lint:fix": "prettier --write --loglevel=warn . && pnpm lint -- --fix",
```

We should also change this to a `;`, just to make sure both parts run indepedently. Other than that, this was much needed.
```suggestion
    "lint:fix": "prettier --write --loglevel=warn .; pnpm lint -- --fix",
```